config/jacoco.gradle

```diff
@@ -1,21 +1,14 @@
 apply plugin: 'jacoco'
 
 jacoco {
-    toolVersion = "$jacoco_core_version"
+    toolVersion = "0.8.3"
```

Comment:
please explain the reason why we have to downgrade this jacoco ver too 🙏 I can't get it from PR description https://github.com/nimblehq/android-templates/pull/41
Also, why do we need to remove ver definition from build.gradle?

Reply:
Just reverting the changes from #41